tests/unit/plugins/modules/test_ome_smart_fabric_uplink_info.py

```diff
@@ -998,57 +1092,64 @@ def test_get_fabric_id_from_name_HTTPError_error_case(self, params, ome_default_
                                                                    StringIO(json_str))
         f_module = self.get_module_mock()
         with pytest.raises(Exception) as exc:
-            self.module.get_fabric_id_from_name(f_module, ome_connection_mock, params.get('fabric_name'))
+            self.module.get_fabric_id_from_name(
+                f_module, ome_connection_mock, params.get('fabric_name'))
         assert exc.value.args[0] == error_msg
 
     @pytest.mark.parametrize("params", [{"fabric_id": "f1", "uplink_id": "u1"}])
     def test_get_uplink_details_HTTPError_error_case(self, params, ome_default_args, mocker,
                                                      ome_connection_mock):
         json_str = to_text(json.dumps({"info": "error_details"}))
-        error_msg = "Unable to retrieve smart fabric uplink information with uplink ID {0}.".format(params.get('uplink_id'))
+        error_msg = "Unable to retrieve smart fabric uplink information with uplink ID {0}.".format(
+            params.get('uplink_id'))
         ome_connection_mock.invoke_request.side_effect = HTTPError('http://testdell.com', 404,
```

Comment:
use https here, otherwise will get sonar issue, I know it's no use, but we can avoid that by simply making it from http to https